src/runtime/batch_monitor.py

```
@@ -52,12 +52,21 @@ def __init__(self):
             raise ValueError(
                 "ANTHROPIC_API_KEY environment variable required for batch automation. "
                 "Get your key at: https://console.anthropic.com/settings/keys"
             )
-        self.client = anthropic.Anthropic(api_key=api_key)
+
+        base_url = os.environ.get("ANTHROPIC_BASE_URL")
+        if not base_url:
+            raise ValueError(
+                "ANTHROPIC_BASE_URL environment variable required for batch automation. "
+                "Get your key at: https://console.anthropic.com/settings/keys"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)